hangupsbot/plugins/getadmins.py

```diff
@@ -0,0 +1,17 @@
+# Original Author: kilr00y@esthar.net
+
+import plugins
+
+from hangups.user import UserList
+
+def _initialise(bot):
+    plugins.register_user_command(["admins"])
+
+def admins(bot,event,*args):
+    admin_list=bot.get_config_option('admins')
+    text="<b><u>List of Admins</u></b><br />"
+    for admin_id in admin_list:
+        user_object = UserList.get_user(bot._user_list,(admin_id,admin_id))
```

Comment:
hangouts forgets users, [Permanent Memory: get_hangups_user()](https://github.com/hangoutsbot/hangoutsbot/wiki/Permanent-Memory#retrieve-most-complete-single-user-record) resolves this by maintaining records independent of the hangups implementation. all instance of `hangups.user.UserList.get_user` usage should be removed and replaced with this.

I also like to point out (although this won't be useful in this specific use case that the `user_data` key in `memory.json` contains the record of every single encountered by the bot and is held long-term so that it won't have to perform additional server lookups to "complete" its memory.